src/test/kotlin/com/igorwojda/integer/addupto/challenge.kt

```diff
@@ -4,7 +4,7 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun addUpTo(n: Int): Int {
-    TODO("not implemented")
+    return (1..n).sum()
```

Comment:
nitpick: As a side implementation, do you remember the math formula for computing the sum of consecutive integers?

Reply:
n*(n+1)/2 de la 1 la n
sau n*(n+i)/2 daca e de la i la n